src/Jackett.Common/Indexers/HDOlimpo.cs

```diff
@@ -93,20 +94,17 @@ private async Task DoLogin()
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
             var includePremium = ((BoolItem)configData.GetDynamic("IncludePremium")).Value;
-
-            var pairs = new Dictionary<string, string>
-            {
-                {"freetorrent", "false"},
-                {"ordenar_por", "created_at"},
-                {"orden", "desc"},
-                {"titulo", query.GetQueryString()}
-            };
-
             var cats = MapTorznabCapsToTrackers(query);
-            var category = cats.Count == 1 ? cats.First() : "0";
-            pairs.Add("categoria", category);
 
-            var boundary = "---------------------------" + (DateTime.UtcNow.Subtract(new DateTime(1970, 1, 1))).TotalSeconds.ToString(CultureInfo.InvariantCulture).Replace(".", "");
+            var pairs = new Dictionary<string, string>
+                {
+                    {"freetorrent", "false"},
+                    {"ordenar_por", "created_at"},
+                    {"orden", "desc"},
+                    {"titulo", query.GetQueryString()},
+                    {"categoria", cats.Count == 1 ? cats.First() : "0"}
+                };
+            var boundary = "---------------------------" + DateTimeUtil.DateTimeToUnixTimestamp(DateTime.UtcNow).ToString();
```

Comment:
I'm not sure about this change, the toString and Replace part. I copied this code from other part of the Jackett code and I know it's working. I prefer to do it in other PR if you want.

Reply:
I know this is already resolved, but due to a conflict in rebasing after the extensions this part came to my attention again. According to https://stackoverflow.com/questions/2305218/what-is-the-boundary-parameter-in-an-http-multi-part-post-request the boundary string doesn't have to be anything in particular except for the dashes. The remainder is a random string that just has to be the same in all places for a specific request.

Reply:
As I said, just copied and tested => https://github.com/Jackett/Jackett/search?q=%28DateTime.UtcNow.Subtract%28new+DateTime%281970%2C+1%2C+1%29%29&unscoped_q=%28DateTime.UtcNow.Subtract%28new+DateTime%281970%2C+1%2C+1%29%29